packages/ens-deployments/src/ens-test-env.ts

```diff
@@ -2,34 +2,34 @@ import { mergeAbis } from "@ponder/utils";
 import { anvil } from "viem/chains";
 
 import { ETHResolverFilter } from "./lib/filters";
-import type { ENSDeploymentConfig } from "./lib/types";
+import type { ENSDeployment } from "./lib/types";
 
-// Subregistry ABIs for direct subnames of 'eth' in ens-test-env
-import { BaseRegistrar as eth_BaseRegistrar } from "./abis/eth/BaseRegistrar";
-import { EthRegistrarController as eth_EthRegistrarController } from "./abis/eth/EthRegistrarController";
-import { EthRegistrarControllerOld as eth_EthRegistrarControllerOld } from "./abis/eth/EthRegistrarControllerOld";
-import { LegacyPublicResolver as eth_LegacyPublicResolver } from "./abis/eth/LegacyPublicResolver";
-import { NameWrapper as eth_NameWrapper } from "./abis/eth/NameWrapper";
-import { Registry as eth_Registry } from "./abis/eth/Registry";
-import { Resolver as eth_Resolver } from "./abis/eth/Resolver";
+// ABIs for Root Datasource
+import { BaseRegistrar as eth_BaseRegistrar } from "./abis/root/BaseRegistrar";
+import { EthRegistrarController as eth_EthRegistrarController } from "./abis/root/EthRegistrarController";
+import { EthRegistrarControllerOld as eth_EthRegistrarControllerOld } from "./abis/root/EthRegistrarControllerOld";
+import { LegacyPublicResolver as eth_LegacyPublicResolver } from "./abis/root/LegacyPublicResolver";
+import { NameWrapper as eth_NameWrapper } from "./abis/root/NameWrapper";
+import { Registry as eth_Registry } from "./abis/root/Registry";
+import { Resolver as eth_Resolver } from "./abis/root/Resolver";
 
 /**
- * The "ENS deployment" configuration for 'ens-test-env'.
+ * The ens-test-env ENSDeployment configuration
```

Comment:
```suggestion
 * The ens-test-env ENSDeployment
```

Since it is no longer an ENSDeploymentConfig